src/binaryen-c.cpp

```diff
@@ -1573,6 +1575,21 @@ BinaryenExpressionRef BinaryenMemoryFill(BinaryenModuleRef module,
   }
   return static_cast<Expression*>(ret);
 }
+BinaryenExpressionRef BinaryenPush(BinaryenModuleRef module,
+                                   BinaryenExpressionRef value) {
+  auto* ret = Builder(*((Module*)module)).makePush((Expression*)value);
```

Comment:
I think there is an extra set of parens around `(Module*)module`.

Reply:
You're right. By the way I actually copy-pasted from other functions, so maybe we can remove all of them.